docs/overview/project.rst

```diff
@@ -0,0 +1,128 @@
+The flexibility provided allows to duplicate, delete, ignore and create content within the constraints of the schema some of which are conditional and will thus react to certain inputs in specific fields.
+
+Every schema change is versioned, ensuring that the schema can be adapted to work with new requirements at any moment in time without breaking older analysis records.
+
+Depending on the preference and work environment of the researcher, analysis information can be created and edited through a `Submission Form`_ on the web interface or via the `REST API`_.
+
+Submission Form
+~~~~~~~~~~~~~~~
+
+Submission forms are the graphical representation of the JSON schemas described in the last section. They allow manual editing of each field as well as inserting JSON conforming to the JSON schema of this part of the analysis form. Auto-complete and auto-fill will support editing fields for which information exists in one of the connected databases. See the section on `Connectivity`_ for more information.
+There are tutorials to help you :doc:`create an analysis <../tutorials/create>` using a submission form.
+
+Visit the :doc:`glossary <../reference-guide/schema-reference>` for more information on the terminologies used in the form.
+
+REST API
+~~~~~~~~
+
+There will be an API to enable direct content upload without using the submission form. A prototype will be available in the near future. If you want to track development you can check it `here <../status.html#capture>`_. A tutorial on using the API will be provided :doc:`here <../tutorials/create-api>` and a reference guide will be found :doc:`here <../reference-guide/api>`.
+
+Connectivity
+~~~~~~~~~~~~
+
+To reduce user effort and ensure accuracy of added information, we are connecting to collaboration databases and systems containing analysis information. This allows us to auto-complete and auto-fill most of an analysis record as soon as it is created, given that the content exists in the databases we link to, as well as to keep the content updated and synchronized.
+
+You can check the status of current connections `here <../status.html#capture>`_. For more details on how autofill works you can go to the tutorial for :doc:`creating an analysis <../tutorials/create>`.
+
+Additionally, there is a possibility for uploading files (e.g. configuration files) as well as providing a URL from which the files are copied and stored automatically.
+
+.. TODO link to file upload tutorial
+
+Versioning
+~~~~~~~~~~
+
+Upon creation of an analysis on CAP it will be assigned a unique identifier. Every time the analysis is edited the new version will be stored as an update to the previous version of the analysis through the identifier system. This will enable references to intermediate analysis steps in the analysis notes, allow keeping track of the analysis and monitoring as well as undoing changes.
```

Comment:
@pamfilos @tiborsimko @daslerr I would like to know a bit more about the PID management in Invenio and how to expose it to users best during the next weeks.